mseed/msrecord.go

```diff
@@ -133,3 +134,25 @@ func (m *MSRecord) SrcName(quality int8) string {
 	C.msr_srcname((*_Ctype_struct_MSRecord_s)(m), csrcname, C.flag(quality))
 	return C.GoString(csrcname)
 }
+
+// cleanString removes all non UTF8, spaces, and null termination
+// characters from s.
+func cleanString(s string) string {
+	if !utf8.ValidString(s) {
```

Comment:
strictly speaking I don't know.  Maybe none?  I do need to save into a UTF8 encoded DB and originally the null terminators were the issue.  Maybe total overkill to do the UTF8 checking?